management/connection.go

```diff
@@ -70,7 +70,7 @@ type Connection struct {
 	// connection name will be added as realm.
 	Realms []interface{} `json:"realms,omitempty"`
 
-	Metadata *interface{} `json:"metadata,omitempty"`
+	Metadata map[string]string `json:"metadata,omitempty"`
```

Comment:
I think this should be `map[string]interface{}`, to allow values other than strings.

Reply:
@paddycarey thanks for the feedback.

I just experimentally verified against Auth0's management API that it will only accept string values.  I tried setting objects and numbers and it failed validation and returned a 400 in each case.  

Auth0 hasn't done a good job of documenting this particular field but as far as I can determine it will only allow string values.

It is also worth noting that the data type for ClientMetadata on the client resource is also defined as map[string]string in this SDK and not map[string]interface{} 

So I think map[string]string is probably correct in this specific case